tests/integration/test_header_links.py

```diff
@@ -76,6 +76,44 @@ def test_links_in_header_when_valid_session(self):
         self.assert_sign_out_link_exist()
         self.assert_help_link_exist()
 
+    def test_links_in_header_when_no_session_but_cookie_exists(self):
+        # Given
+        self.launchSurvey("test_thank_you")
+        self.assertInUrl("questionnaire/did-you-know/")
+        self.saveAndSignOut()
+
+        # When
+        self.assertStatusCode(302)
+        self.get("questionnaire/")
+
+        # Then
+        self.assertInUrl("questionnaire/")
+        cookie = self.getCookie()
+        self.assertIsNotNone(cookie.get("theme"))
+        self.assertEqual(cookie.get("theme"), "default")
```

Comment:
The first assert is covered by the second, so the `self.assertIsNotNone` can be removed.